Automattic/wp-calypso
# State: Remove unused Jetpack Jumpstart state, components and utilities 🗑

Almost a year and a half ago, while implementing the Jetpack settings in Calypso we anticipated to take advantage of the Jumpstart feature. We implemented all the necessary Redux infrastructure back then, but we never actually got to use it. So, we're safe to remove it, and that is what this PR does.

To test:
* Checkout this branch
* Verify that none of the removed state (actions, reducer, selectors), query component and wpcom-undocumented methods are in use.
* Verify all state tests still pass:

```
npm run test-client client/state
```
